webflow-hacks/interactions-hacks/38.ts

```diff
@@ -0,0 +1,17 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const form: HTMLElement = document.getElementsByClassName('hack38-form')[0] as HTMLElement;
+    const successTrigger: HTMLElement =document.getElementsByClassName('hack38-form-success-trigger')[0] as HTMLElement;;
```

Comment:
Same here, when we publish JS snippets for users to copy/paste them, we should make them as safe as possible to avoid runtime errors.

```typescript
const form = document.querySelector<HTMLElement>('.hack38-form');
const successTrigger = document.querySelector<HTMLElement>(
  '.hack38-form-success-trigger'
);

if (!form || !successTrigger) return;
```